eng/testing/tests.mobile.targets

```diff
@@ -143,23 +143,27 @@
              AssemblyFile="$(WasmAppBuilderTasksAssemblyPath)" />
 
   <Target Condition="'$(TargetOS)' == 'Browser'" Name="BundleTestWasmApp">
+    <PropertyGroup>
```

Comment:
This is already done at https://github.com/dotnet/runtime/blob/9765835057496db6ec2c1e5fb7a7610f628915ef/eng/testing/tests.props#L31